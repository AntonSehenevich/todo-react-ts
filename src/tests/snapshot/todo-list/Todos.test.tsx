import { Provider } from 'react-redux'
import renderer from 'react-test-renderer'
import Todos from '../../../components/todos'
import setupStore from '../../../store'

test('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={setupStore()}>
        <Todos />
      </Provider>
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
