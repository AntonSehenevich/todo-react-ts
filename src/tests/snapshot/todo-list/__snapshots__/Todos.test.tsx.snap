// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="container"
    data-testid="add-todo"
  >
    <input
      className="input"
      onBlur={[Function]}
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="Something to do?"
      type="text"
      value=""
    />
  </div>
</div>
`;
